### ISSUE TITLE
Add 'microhttpd' package

### ISSUE THREAD

**Issue by chfast** [ruslo/hunter#951]
microhttpd library: https://www.gnu.org/software/libmicrohttpd/
Git: https://gnunet.org/git/libmicrohttpd.git

They use autotools on Unixes and Visual Studio projects on Windows. The CMake support was proposed, but rejected -- they don't want to maintain it upstream. I tried to contract the author of the CMake script, but no response yet. See http://lists.gnu.org/archive/html/libmicrohttpd/2017-07/msg00007.html

Can Hunter support autotools and VS project configs?

**Comment by ruslo** [comment]
> Can Hunter support autotools and VS project configs?

Yes, can but if we already have CMakeLists.txt it will be much easier to create a fork. Custom schemes are huge pain and really should be used if there are no alternatives.

**Comment by chfast** [comment]
Can you create microhttpd repo then. I will push the code there.

**Comment by ruslo** [comment]
Done: https://github.com/hunter-packages/microhttpd
With branch `hunter-0.9.55`.
